tests/integration/favorites_tools.js

```diff
@@ -1,5 +1,7 @@
+/* global PANEL_ANIMATION_DELAY_MS */
```

Comment:
I'm not sure this is the best way to store our variables, as the number could grow over the time.
Should'nt we share a module and import it into the relevant tests files ?

Reply:
Same problem you met lately: this is something defined only in CSS, not in JS.
Maybe there are alternatives and I'm open to it, as long as it doesn't get overkill :) (here it's just for the tests)

For example, we can wait for the transition without knowing its actual duration, but it needs some work :)
https://stackoverflow.com/questions/51547994/wait-for-transition-to-end-in-puppeteer
WDYT?

Reply:
waiting for the transition to complete seems to be the more generic solution. Can we give it a try using the transition event ? And maybe define an util function in our helpers ?

Reply:
Ok, did it. At first I thought it didn't work, but it does :) You just have to listen to the right event, in this case it wasn't `transitionend` but `animationend`, because of the way this animation is declared in CSS.